src/Microsoft.DotNet.Build.Tasks.Feed/BuildManifest/UpdateOrchestratedBuildManifest.cs

```diff
@@ -92,21 +102,41 @@ public override bool Execute()
         {
             try
             {
-                await client.PushChangeAsync(
+                var location = new BuildManifestLocation(
                     new GitHubProject(VersionsRepo, VersionsRepoOwner),
                     $"heads/{VersionsRepoBranch}",
-                    VersionsRepoPath,
-                    OrchestratedBuildId,
-                    manifest =>
-                    {
-                        foreach (var update in ManifestUpdates ?? Enumerable.Empty<ITaskItem>())
+                    VersionsRepoPath);
+
+                await client.PushChangeAsync(
+                    new BuildManifestChange(
+                        location,
+                        CommitMessage,
+                        OrchestratedBuildId,
+                        SemaphoreNames,
+                        manifest =>
                         {
-                            ApplyUpdate(manifest, update);
-                        }
-                    },
-                    SemaphoreNames,
-                    PushOrchestratedBuildManifest.CreateUploadRequests(SupplementaryFiles),
-                    CommitMessage);
+                            foreach (var update in ManifestUpdates ?? Enumerable.Empty<ITaskItem>())
+                            {
+                                ApplyUpdate(manifest, update);
+                            }
+                        })
```

Comment:
I might be mistaken...but is there a comma missing here?  For readability it might be good to break out the "new ..." into a separate statement if possible.

Reply:
Maybe what you're not seeing is that this is a constructor with args and an object initializer block. Agree pulling things out will probably clear it up.